multicast/exceptions.py

```
@@ -431,6 +433,82 @@
+			True
+			>>> # Verify boundary exceptions are properly handled
+			>>> try:
+			...     validate_exit_code(999)
+			...     success = False
+			... except ValueError:
+			...     success = True
+			>>> success
+			True
+	"""
+	if not isinstance(code, int) or code < 0 or code > 255:
+		raise ValueError("Exit code must be an integer between 0 and 255")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)